gateway/handlers/proxy.go

```diff
@@ -140,7 +141,8 @@ func invokeService(w http.ResponseWriter, r *http.Request, metrics metrics.Metri
 	contentType := r.Header.Get("Content-Type")
 	fmt.Printf("[%s] Forwarding request [%s] to: %s\n", stamp, contentType, url)
 
-	request, err := http.NewRequest("POST", url, bytes.NewReader(requestBody))
+	method := r.Method
+	request, err := http.NewRequest(method, url, bytes.NewReader(requestBody))
```

Comment:
We can inline r.Method

Reply:
Yes, that is true, if you would prefer that I can make that change :smile: